executor/common.h

```diff
@@ -704,6 +704,12 @@ static void loop(void)
 				executed_calls = now_executed;
 				last_executed = now;
 			}
+
+			// Globs can be slow. But running fuzzing without them can lead to coverage degradation
+			// Especially considering glob failure is not a fatal error and can go unnoticed
+			if (request_type == rpc::RequestType::Glob)
```

Comment:
This needs to be combined with the `request_type` check below and the comment needs updates.
But I wouldn't do no timeout at all. Is there some reasonable max time for this operation? Minute? 10 minutes?
If it really hangs, the failure will be very unpleasant. Imagine, tomorrow somebody comes to us saying "manager hangs dead with no output whatsoever, debug it". What do we?

Reply:
Fair enough, for some reason I assumed there should be a timeout on the manager side.

I fixed it to just be an increased timeout (120 seconds seams reasonable - if you still timeout with that, you are probably doing something wrong). I've also changed some comments and merged the conditions, they should be more consistent now.